tests/IceRpc.Slice.Tests/ExceptionTests.slice

```diff
@@ -25,7 +25,6 @@ exception EmptyException {}
 
 interface SliceExceptionOperations {
     opThrowsMultipleExceptions() throws (MyException, EmptyException)
-
```

Comment:
I think it's a bit cleaner to have them all separated by a newline, or all by none.

Reply:
Fixed